tailcfg/tailcfg.go

```diff
@@ -2250,6 +2250,9 @@ const (
 
 	// NodeAttrLogExitFlows enables exit node destinations in network flow logs.
 	NodeAttrLogExitFlows NodeCapability = "log-exit-flows"
+
+	// NodeAttrAutoExitNode enables auto exit nodes.
```

Comment:
Can you flesh this out? Does it make an option available or does it immediately switch to one?

Also: automatic

Reply:
it makes the option available